src/mpags-cipher.cpp

```diff
@@ -87,70 +60,53 @@ int main(int argc, char* argv[])
     // Initialise variables
     char inputChar{'x'};
     std::string inputText;
+    std::string outputText;
+    std::ifstream in_file;
+    std::ofstream out_file;
 
     // Read in user input from stdin/file
     // Warn that input file option not yet implemented
     if (!inputFile.empty()) {
-        std::cerr << "[warning] input from file ('" << inputFile
-                  << "') not implemented yet, using stdin\n";
-    }
-
-    // loop over each character from user input
-    while (std::cin >> inputChar) {
-        // Uppercase alphabetic characters
-        if (std::isalpha(inputChar)) {
-            inputText += std::toupper(inputChar);
-            continue;
+        in_file.open(inputFile);
+        bool ok_to_read = in_file.good();
+        if (!ok_to_read){
+            std::cerr << "[Warning] input from file ('" << inputFile
+                      << "') not implemented yet, remove -i and type letters in for command line interface.\n"
+                      << std::endl;
```

Comment:
It isn't that reading from `inputFile` isn't implemented, but rather that the file specified either doesn't exist or cannot be opened for some reason (e.g. no read permission).